Gemfile

```diff
@@ -1,2 +1,4 @@
 source "https://rubygems.org"
 gem 'github-pages'
+
+gem "webrick", "~> 1.8"
```

Comment:
what is this for?

Reply:
Ah sorry I can get rid of it, that was something Ruby made me add to host it on my machine